The HTTPInjectOptions typescript interface does not define the 'cookies' property

## 🐛 Bug Report

The HTTPInjectOptions typescript interface should define the `cookies` property

## To Reproduce

The following code can not be compiled with typescript (sorry, the code cannot be run as is, but describes the problem well):
```typescript
    const fastify: FastifyInstance = ...
    const response: HTTPInjectResponse = await fastify
        .inject({
            method: 'POST',
            url: '/some-url',
            payload: { 'some': 'payload' },
            cookies: { foo: 'bar' },
        })
```

## Expected behavior

The code should be compilable with typescript, but I get the following error:

```
error TS2345: Argument of type '{ method: "POST"; url: string; payload: {... }; cookies: { foo: string; }; }' is not assignable to parameter of type 'string | HTTPInjectOptions'.
  Object literal may only specify known properties, and 'cookies' does not exist in type 'HTTPInjectOptions'.
```

## Your Environment

- *node version*: 12
- *fastify version*: >=2.12.0
- *os*: Mac

--- a/fastify.d.ts
+++ b/fastify.d.ts
@@ -8,6 +8,7 @@ import * as ajv from 'ajv'
 import * as http from 'http'
 import * as http2 from 'http2'
 import * as https from 'https'
+import * as LightMyRequest from 'light-my-request'
 
 declare function fastify<
   HttpServer extends (http.Server | http2.Http2Server) = http.Server,
@@ -315,42 +316,6 @@ declare namespace fastify {
     [key: string]: any,
     prefix?: string,
     logSerializers?: Object
-  }
-
-  /**
-   * Fake http inject options
-   */
-  interface HTTPInjectOptions {
-    url: string,
-    method?: HTTPMethod,
-    authority?: string,
-    headers?: DefaultHeaders,
-    query?: DefaultQuery,
-    remoteAddress?: string,
-    payload?: string | object | Buffer | NodeJS.ReadableStream
-    simulate?: {
-      end?: boolean,
-      split?: boolean,
-      error?: boolean,
-      close?: boolean
-    },
-    validate?: boolean
-  }
-
-  /**
-   * Fake http inject response
-   */
-  interface HTTPInjectResponse {
-    raw: {
-      req: NodeJS.ReadableStream,
-      res: http.ServerResponse
-    },
-    headers: Record<string, string>,
-    statusCode: number,
-    statusMessage: string,
-    payload: string,
-    rawPayload: Buffer,
-    trailers: object
   }
 
   /**
@@ -678,12 +643,12 @@ declare namespace fastify {
     /**
      * Useful for testing http requests without running a sever
      */
-    inject(opts: HTTPInjectOptions | string, cb: (err: Error, res: HTTPInjectResponse) => void): void
+    inject(opts: LightMyRequest.InjectOptions | string, cb: (err: Error, res: LightMyRequest.Response) => void): void
 
     /**
      * Useful for testing http requests without running a sever
      */
-    inject(opts: HTTPInjectOptions | string): Promise<HTTPInjectResponse>
+    inject(opts: LightMyRequest.InjectOptions | string): Promise<LightMyRequest.Response>
 
     /**
      * Set the 404 handler

--- a/fastify.d.ts
+++ b/fastify.d.ts
@@ -37,6 +37,10 @@ declare namespace fastify {
 
   type HTTPMethod = 'DELETE' | 'GET' | 'HEAD' | 'PATCH' | 'POST' | 'PUT' | 'OPTIONS'
 
+  // // Keep the original name of the interfaces to avoid braking change
+  interface HTTPInjectOptions extends LightMyRequest.InjectOptions {}
+  interface HTTPInjectResponse extends LightMyRequest.Response {}
+
   interface ValidationResult {
     keyword: string;
     dataPath: string;
@@ -643,12 +647,12 @@ declare namespace fastify {
     /**
      * Useful for testing http requests without running a sever
      */
-    inject(opts: LightMyRequest.InjectOptions | string, cb: (err: Error, res: LightMyRequest.Response) => void): void
+    inject(opts: HTTPInjectOptions | string, cb: (err: Error, res: HTTPInjectResponse) => void): void
 
     /**
      * Useful for testing http requests without running a sever
      */
-    inject(opts: LightMyRequest.InjectOptions | string): Promise<LightMyRequest.Response>
+    inject(opts: HTTPInjectOptions | string): Promise<HTTPInjectResponse>
 
     /**
      * Set the 404 handler

--- a/test/types/index.ts
+++ b/test/types/index.ts
@@ -615,8 +615,17 @@ server.inject({ url: '/test' }, (err: Error, res: LightMyRequest.Response) => {
   server.log.debug(res.payload)
 })
 
+// http injections with the fastify types
+server.inject({ url: '/test' }, (err: Error, res: fastify.HTTPInjectResponse) => {
+  server.log.debug(err)
+  server.log.debug(res.payload)
+})
+
 server.inject({ url: '/testAgain' })
   .then((res: LightMyRequest.Response) => console.log(res.payload))
+
+server.inject({ url: '/testAgain' })
+  .then((res: fastify.HTTPInjectResponse) => console.log(res.payload))
 
 server.setSchemaCompiler(function (schema: object) {
   return () => true

--- a/test/types/index.ts
+++ b/test/types/index.ts
@@ -6,6 +6,7 @@ import * as fastify from '../../fastify'
 import * as http from 'http'
 import * as http2 from 'http2'
 import { readFileSync } from 'fs'
+import * as LightMyRequest from 'light-my-request'
 
 // were importing cors using require, which causes it to be an `any`. This is done because `cors` exports
 // itself as an express.RequestHandler which is not compatible with the fastify TypeScript types
@@ -609,13 +610,13 @@ server.listen({
 }).then((address: string) => console.log(address))
 
 // http injections
-server.inject({ url: '/test' }, (err: Error, res: fastify.HTTPInjectResponse) => {
+server.inject({ url: '/test' }, (err: Error, res: LightMyRequest.Response) => {
   server.log.debug(err)
   server.log.debug(res.payload)
 })
 
 server.inject({ url: '/testAgain' })
-  .then((res: fastify.HTTPInjectResponse) => console.log(res.payload))
+  .then((res: LightMyRequest.Response) => console.log(res.payload))
 
 server.setSchemaCompiler(function (schema: object) {
   return () => true

